DvdLib/DvdLib.csproj

```
@@ -13,7 +13,8 @@
     <TargetFramework>net5.0</TargetFramework>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AnalysisMode>AllDisabledByDefault</AnalysisMode>
+    <Nullable>Disable</Nullable>
```

`enable` is lowercase in the Directory.Build.props, maybe this should be too to be consistent?
```suggestion
    <Nullable>disable</Nullable>
```